src/field.rs

```diff
@@ -624,6 +650,20 @@ pub fn random_vector<F: FieldElement>(len: usize) -> Result<Vec<F>, PrngError> {
         .collect())
 }
 
+/// Generate a vector of uniform random field elements.
```

Comment:
This is https://github.com/cjpatton/vdaf/pull/30, right? Can you give this a more meaningful name, to help me keep this straight?